PrReviewCode/Program.cs

```diff
@@ -4,6 +4,8 @@ public class Program
 {
     public static void Main(string[] args)
     {
+        throw new Exception("Test Exception");
```

Comment:
Test exception left in production code. This should be removed before merging.

 **SUGGESTION:**

```suggestion
 // Remove or replace with proper error handling
```